modules/tinymce/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A new `lang` format is now available that can be used with `editor.formatter`, or applied with the `Lang` editor command #TINY-6149
 - Added new plugin commands: `mceEmoticons` (emoticons), `mceWordCount` (wordcount), and `mceTemplate` (template) #TINY-7619
 - Added a new `tablerowheader` toolbar button and menu item to toggle row cells to and from header #TINY-7478
+- Added a new `tablecolheader` toolbar button and menu item to toggle row cells to and from header #TINY-7482
```

Comment:
```suggestion
- Added a new `tablecolheader` toolbar button and menu item to toggle column cells header state #TINY-7482
```
Maybe? This at the very least needs to say column, not row.

Reply:
This still reads really strangely (so does the row one for that matter). I don't think we need the `to and from` part when we're already saying `toggles`

Reply:
```
- Added a new `tablecolheader` toolbar button and menu item to toggle the header state of column cells #TINY-7482
```
This is probably better than my original suggestion.